src/MarkerClusterGroup.js

```diff
@@ -260,6 +260,9 @@ L.MarkerClusterGroup = L.FeatureGroup.extend({
 				m = layersArray[i];
 				this._arraySplice(this._needsClustering, m);
 				npg.removeLayer(m);
+				if (this.hasLayer(m)) {
```

Comment:
Add a comment on this detailing why it is done please, it looks a bit weird when you look at it standalone.